joomla/joomla-cms
# [4.0] Override-Manager fix

# Summary of Changes

At the moment you can not see in the template manager what function the Switches have. It looks like this to me. 

![Templates  Customise  Cassiopeia    test   Administration(3)](https://user-images.githubusercontent.com/9974686/77068560-1e9a1e00-69e7-11ea-8dc7-b09550896316.png)

I have also corrected the alignment. For `ltr` it was only needed to remove a margin. The alignment is OK. For `rtl` I added the correct alignment.

The label is not inserted. We would have to use 
`echo $this→form→renderField('show_core');`
for this.

I tried to display the label. However, it is difficult to position the switcher with label correctly at this point. Therefore I think it is good to only display the `input` and to adapt the description text.


### Testing Instructions
1. Create an override in the Template Manager and open it for editing. You see two switches. You don't know what these switches do.

2. Apply this patch

3. Switch back to the template manager and see that the description text of the switches now shows the current status. You now know what switch is responsible for.


### Expected result
You can see the function that a switch performs based on the label or description.

![قالب ها  اختصاصی سازی قالب  Cassiopeia    test   مدیریت](https://user-images.githubusercontent.com/9974686/77068848-9405ee80-69e7-11ea-8557-f72ab3cb1a5d.png)

![Templates  Customise  Cassiopeia    test   Administration(2)](https://user-images.githubusercontent.com/9974686/77068850-949e8500-69e7-11ea-8dfd-9016bc9b48e4.png)


### Actual result
You see switches but you do not know what the switch `hide` or `show`.


### Additional information

Because of  https://github.com/joomla/joomla-cms/blob/9f68e6e9950fbdb07cbc9ce5a8e90ff676dec845/administrator/templates/atum/scss/template-rtl.scss#L209 we need the change in the CSS-file.

This is a redo of parts of this PR: https://github.com/joomla/joomla-cms/pull/26359/
